zerver/tests/test_auth_backends.py

```diff
@@ -1891,6 +1893,97 @@ def test_validate_state(self) -> None:
             self.assertIn('login', result.url)
             m.assert_called_with("Sign in with Apple failed: bad state token.")
 
+class AppleAuthBackendNativeFlowTest(AppleAuthMixin, SocialAuthBase):
+    '''
+    This class is to test the native flow we continue with the
+    data sent as GET request from mobile app.
+    '''
```

Comment:
Tests coverage isn’t there because __unittest_skip = False isn’t set here. Will fix it.

Reply:
Fixed it! could you review again @mateuszmandera :slightly_smiling_face: